test/e2e/load/load.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2021 ScyllaDB
+
+package load
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	"github.com/gocql/gocql"
+)
+
+type Load struct {
+	cluster           *gocql.ClusterConfig
```

Comment:
Consider using https://github.com/scylladb/gocqlx

Reply:
considered and used :stuck_out_tongue: